.circleci/config.yml

```diff
@@ -37,6 +37,13 @@ commands:
             else
               python -m venv venv
               . venv/bin/activate
+              # wheel is in requirements.txt, but without doing this first, the
+              # below pip3 install is very slow, with lots of warnings like:
+              #
+              #   Using legacy 'setup.py install' for Flask-JWT-Extended, since package 'wheel' is not installed.
+              #
+              # afterwards, it is merely pretty slow!
+              # pip install --upgrade wheel  # TODO see if this is still needed
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker